examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
```

Comment:
Agreed, this has caused confusion already in reviewers.  I added comments that specifically state  that a hostPath volume should be used only and exclusively for development and testing.